Rename default json endpoint (#239)
quantumblacklabs/kedro-viz: CONTRIBUTING.md

DATA=random npm start
```

> There are several different data sources available. By default in production, the app asynchronously loads JSON from the `/api/main` endpoint, and you can replicate this in development by placing a dataset at `/public/api/main`. Note that `main` is the name of the json file, but should not have an extension. Alternatively, you can use one of the mock unit-testing/demo datasets, or pseudo-random data procedurally-generated on page load, which is often useful for local development.

Reviewer: ```suggestion
There are several different data sources available. By default in production, the app asynchronously loads JSON from the `/api/main` endpoint. You can replicate this in development by placing a dataset in `/public/api/main`, using `main` as the name of the JSON file, without an extension. Alternatively, you can use one of the mock unit-testing/demo datasets, or pseudo-random data procedurally-generated on page load, which is often useful for local development.
```
